### ISSUE TITLE
Notebook 9 - Rank-Deficient Least-Squares Reaction Rates

### ISSUE THREAD

**Issue by joeegitto22** [dpploy/chen-3170#20]
This notebook definitely contained a lot of vital information that was used throughout the rest of the course. Definitely make sure that at this point students are comfortable with this information in lecture.
I feel that we did not discuss a lot about the sub-mechanisms in lecture and during the lab period and may be why some people struggled with this section in the midterm project. The extra-help class that you held really helped to clarify everything and put all of the pieces together as you were actively writing the code with us. I feel as though the importance of the plots at the end of this notebook were not explained in lecture.

**Comment by dealmeidavf** [comment]
Right. I am writing this note to myself. In the future, the basic solution will be replaced by an augmented LS solution that minimizes the norm of the solution vector in addition to the residual. This will require more linear algebra content on orthogonal factorization and I hope this additional content will fit in the course once basic python programming is covered in an earlier course such as Introduction to Chemical Engineering.
